Resources/Prototypes/Entities/Structures/Doors/Shutter/blast_door.yml

```diff
@@ -43,3 +53,46 @@
     airBlocked: false
   - type: RadiationBlocker
     enabled: false
+
+- type: entity
+  id: BlastDoorFrame
+  parent: BaseStructureDynamic
+  name: blast door frame
+  description: This one says 'BLAST DONGER'.
+  components:
+  - type: Sprite
+    sprite: Structures/Doors/Shutters/blastdoor.rsi
+    state: assembly
+  - type: InteractionOutline
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: Metallic
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 300
+      behaviors:
+      - !type:DoActsBehavior
+        acts: ["Destruction"]
+  - type: Physics
+    bodyType: Dynamic
```

Comment:
bodyType: Static
Its should fix error checks

Reply:
My mistake, I forgot to change it